README.md

```diff
@@ -290,13 +280,13 @@ Training a model takes around 4 days for 100 epochs (35 min per epoch), includin
 | VICReg         | Res50    |        256 |    100 |        63.0 |          73.7 |     46.3 |      [link](https://tensorboard.dev/experiment/qH5uywJbTJSzgCEfxc7yUw) |       [link](https://lightly-ssl-checkpoints.s3.amazonaws.com/imagenet_resnet50_vicreg_2023-09-11_10-53-08/pretrain/version_0/checkpoints/epoch%3D99-step%3D500400.ckpt) |
 
 *\*We use square root learning rate scaling instead of linear scaling as it yields
-better results for smaller batch sizes. See Appendix B.1 in [SimCLR paper](https://arxiv.org/abs/2002.05709).*
+better results for smaller batch sizes. See Appendix B.1 in the [SimCLR paper](https://arxiv.org/abs/2002.05709).*
 
 
 
-### ImageNette
+### Imagenette
```

Comment:
Maybe we could just drop imagenette and cifar here and put a link to the docs instead? This would make the readme shorter and easier to read. I don't think anyone is particularly interested in imagenette results.